applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -178,6 +170,14 @@ config SD_CARD_PLAYBACK_THREAD_PRIORITY
 	int "Priority for the SD card playback thread"
 	default 7
 
+module = MODULE_SD_CARD
```

Comment:
This will only be ran if `SD_CARD_PLAYBACK` is enabled, but the SD card module is included with `NRF5340_AUDIO_SD_CARD_MODULE`, so it should be moved out of the if. (Not sure if we have a use case for the SD card without the playback module tho)